AsyncDisplayKit/Layout/ASStackLayoutSpec.h

```diff
@@ -35,7 +35,7 @@
  */
 @interface ASStackLayoutSpec : ASLayoutSpec
 
-/** Specifies the direction children are stacked in. */
+/** Specifies the direction children are stacked in. It becomes immutable after either horizontal or vertical alignment was set */
```

Comment:
Can we change the behavior so this property can only be set once?  Then the comment would also be more clear: "The stack direction may only be set once, and is then immutable" (or could be even shorter).

I think there is no valid use case for changing the stack direction, and this restriction helps ensure the correctness of the other features with less complexity.